release-notes/opensearch-security.release-notes-2.1.0.0.md

```diff
@@ -0,0 +1,33 @@
+## 2022-06-30 Version 2.1.0.0
+
+Compatible with OpenSearch 2.1.0
+
+### Enhancements
+* Delegate to NettyAllocator.getAllocator() for ByteBufAllocator instead of hard-coding PooledByteBufAllocator. ([#1396](https://github.com/opensearch-project/security/pull/1396))
+* Tenant Permissions : added the possibility to specify tenants via parameter ([#1813](https://github.com/opensearch-project/security/pull/1813))
+* JWT: validate issuer and audience ([#1780](https://github.com/opensearch-project/security/pull/1780), [#1781](https://github.com/opensearch-project/security/pull/1781)) ([#1785](https://github.com/opensearch-project/security/pull/1785))
+
+### Refactoring
+* Remove master keywords ([#1886](https://github.com/opensearch-project/security/pull/1886))
+
+### Maintenance
```

Comment:
IMO we should delete this section, our customers that read release notes aren't interested in version bumping or release note related pull requests - if they were they should check our our git log instead.

If we do want to keep this section, please move it to the bottom of the document so we can highlight the bug fixes more prominently

Reply:
makes sense... I have moved it to the bottom for now